src/animate/load.ts

```diff
@@ -167,11 +178,19 @@ export function load(scene: AnimateAsset, optionsOrComplete?: Complete | LoadOpt
                     data = metadata.default;
                 }
             }
-            promises.push(Assets.load({ alias: [id], src: basePath + assets[id], data }).then((loadedAsset) =>
+            promises.push(Assets.load({ alias: [id], src: basePath + assets[id], data }, progress =>
             {
+                if (optionsOrComplete && typeof optionsOrComplete !== 'function')
```

Comment:
A couple suggestions here:
* Save a local variable like `complete` up above.
* Since we already have a callback for each asset, instead of making a new function for each asset to pass to Assets, just do our progress calculation where you are doing `loadedAssets++` (which will also ensure that the math is correct).

Reply:
Done